release/serve_tests/workloads/microbenchmarks.py

```diff
@@ -260,6 +332,32 @@ async def _main(
             )
             serve.shutdown()
 
+        if run_streaming:
+            h: DeploymentHandle = serve.run(
+                Benchmarker.bind(
+                    Streamer.options(max_ongoing_requests=1000).bind(
```

Comment:
Should we also do we for intermediate router on the handle?

Reply:
IMO it's not as useful, measuring intermediate router is more to keep track of a more realistic case that users will use (proxy <-> router <-> model). To me the case of handle <-> router <-> model seems unlikely to be a common case, since almost all production use cases should go through proxy.